### PR TITLE
Corrected and slightly rewritten

### PR DESCRIPTION
**Pull request by NNNILabs**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,47 +1,47 @@
 ## What is in this repository?
-Here are complete set of design files, manufacturing data, source files, test results and other resources that I used the process of building nanovoltmeter. This process was not a coincidence, but response to nanovoltmeter challenge by TiN on [his website](https://xdevs.com/article/nvm_comp/)
+Here are the complete set of design files, manufacturing data, source files, test results and other resources that I used in the process of building a nanovoltmeter. This process was not a coincidence, but a response to the nanovoltmeter challenge by TiN on [his website](https://xdevs.com/article/nvm_comp/)
```
Comment (macaba) [inline readme.md]: "Here is the" is the correct use, as "complete set" is singular. "Here are the files" would also be correct.

File: readme.md [hunk 2]
```diff
@@ -1,47 +1,47 @@
...
-Measurement of such as low voltages is useful for many applications, for example resolving small voltage differences in various differential and bridge circuits, measurement of very small resistances - with appropriate current source.
-Commercial nanovoltmeters do exist, like Keithley model 2182A or Keysight 34420a, but with hefty price tag, so DIY design definitely has its place.
+## What is a nanovoltmeter and what is it good for?
+Well, a nanovoltmeter is a voltmeter that has the sensitivity, noise and stability to resolve voltages down to the nanovolts. While many multimeters do have quite sensitive low voltage ranges, the resolution (least significant digit) is usually in the order of hundreds of nanovolts - and value of this last digit is often diminished by the noise and nonlinearity of the meter. So, for proper low signal measurements, having a higher sensitivity is mandatory.
```
Comment (macaba) [inline readme.md]: "down to the nanovolt" is better (singular, not plural).

File: readme.md [hunk 3]
```diff
@@ -1,47 +1,47 @@
...
+While I have been tinkering with DIY test and measurement tools for a while now, I never had an impulse to discover the nanovolt world. Such a trigger happened on September 2nd, 2021, when TiN from xdevs site launched the [nanovolt challenge](https://xdevs.com/article/nvm_comp/). In nutshell, it called for the design of an open-source nanovoltmeter, with parameters comparable to commercial designs and a timeframe of 256 days. That is a non-trivial task, and in my case it was even more so, because I had not only never designed low signal circuits before, but had I never operated a nanovoltmeter. That is quite an unfavorable starting position and jumping into the deep end rather than dipping my toes into the problematics. To ease my mind, I waited a few days to see how others reacted on the [eevblog forum](https://www.eevblog.com/forum/metrology/nanovolt-design-challenge-build-and-show-your-own-nv-meter-in-256-days/?all). I can't say that it encouraged me.
```
Comment (macaba) [inline readme.md:11]: "In a nutshell"

File: readme.md [hunk 4]
```diff
@@ -1,47 +1,47 @@
...
+On the other hand, I spent a few days researching online sources and studying the service manuals of proven test gear from reputable vendors. Too bad they stopped supplying us with schematics in early 90s of the last century; but even that material was very helpful in understanding both what the requirements are for such a meter, as well as the means to achieve it.
 
 ## NVM design, part 1 - design requirements
-Armed with new knowledge I started dissecting the challenge project requirements. 
+Armed with new knowledge I started dissecting the challenge requirements.
```
Comment (macaba) [inline readme.md:15]: "challenging requirements"

File: readme.md [hunk 5]
```diff
@@ -1,47 +1,47 @@
...
+This one calls for either DC or AC power, but it could be combined by having a PSU operating from an appropriate DC voltage (12V sounds like a good starting point) and attach a small AC/DC SMPS to provide power from mains.
 
 > Provide DC Voltage measurement ranges ±100 µV or below and include ±1V and ±10VDC range.
 
-This one looks innocent, but having both 10V and 100uV input on the same jacks implies either switching input amplifiers, or inserting voltage divider into the signal path, preferable by relay. Some older nanovoltmeters (Keithley model 181) had two different inputs - one for higher ranges, another one with low TEMF connector for sensitive ranges.
+This one looks innocent, but having both 10V and 100uV input on the same jacks implies either switching input amplifiers, or inserting a voltage divider into the signal path, preferable through a relay. Some older nanovoltmeters (Keithley model 181) had two different inputs - one for higher ranges, another one with low TEMF connector for sensitive ranges.
```
Comment (macaba) [inline readme.md]: "preferably through a relay"

File: readme.md [hunk 6]
```diff
@@ -1,47 +1,47 @@
...
 
 > Ability to digitize input DC signal with resolution at least 10 nV and noise better than 30 nV peak to peak over at least 0.1-10 Hz bandwidth.
 
-More numeric requirements, nice. 30nVp-p in 10Hz bandwidth is not very relaxed requirement - older meters like Keithley model 181 would not cut it, current models so-so.
+More numeric requirements, nice. 30nVp-p in 10Hz bandwidth is not a very relaxed requirement - older meters like Keithley model 181 would not cut it, current models so-so.
 
 > Have autozero functionality to correct for static offsets.
 
-That is logical requirement. I may do some more effort by introducing autocalibration.
+That is a logical requirement. I put in some more effort and introduce autocalibration.
```
Comment (macaba) [inline readme.md]: "I put in" is not the best fit here. Mixing of current & past tense. 
"I will put in some more effort by introducing autocalibration"

File: readme.md [hunk 7]
```diff
@@ -70,56 +70,56 @@ OK, fair.
...
-So, two PCBs for floating part, one for earthy part, but the backside connectors have to be mounted somehow - here I added another PCB and front panel pushbuttons also needed PCB for mechanical reasons, that resulted in total count of 5 PCBs. I named the PCBs with human names and gave them functionality:
+Designing such an instrument, especially on a DIY base, is a lot of pingponging between mechanical and eletrical design. I could imagine how large the thing would be (say, an A4 page size as a footprint) and I knew I had to cram everything inside. Choosing a larger enclosure would give me much more freedom and other benefits (like easier dissipation of device heat, helping fight TEMF), but would be impractically large. Finding an appropriate enclosure wasn't easy - at first I thought of using G756 I used before in my [DIY SMU](https://github.com/jaromir-sukuba/J-SMU), but I preferred metal enclosures here - because of shielding from electrical interference, and with a steel enclosure even a bit of sheilding from magnetic interference. Metal also helps with heat dissipation. Finally I settled on the 1EP802825 from Modushop.
```
Comment (macaba) [inline readme.md]: s/sheilding/shielding

File: readme.md [hunk 8]
```diff
@@ -70,56 +70,56 @@ OK, fair.
...
-LNA injects ground current into ground net of the circuit, and the current is dependent on LNA input voltage (and therefore LNA output voltage, too). Ground current may cause unwanted voltage shifts between ground potentials of the circuit, especially at high gain operation. To counteract this ground current, amplifier with U121 is setup to source current of opposite polarity into the ground, into areas where LNA resides.
-Bart board is 4-layer, with internal layers being ground nets, for both electrical and thermal reasons.
+Each of the first 10 stage amplifiers in noninverting configuration is brought to a second stage summing amplifier, and all 10 second stage amplifiers are summed into a third stage amplifier. Overall gain of the LNA is 1000.
```
Comment (macaba) [inline readme.md]: Needs a bit of work, maybe something like 
"Each of the first stage gain blocks (with 10 amplifiers in parallel noninverting configuration) is brought to a second stage summing amplifier, and all 10 of the second stage amplifiers are summed into a third, and final, amplifier."

**Review by macaba — COMMENTED** [review]
Overall; very good. I added a few suggestions.

File: readme.md (hunk 4 above)
Comment (jaromir-sukuba) [reply readme.md:15]: I meant the requirements of the challenge, didn't mean the requirements were challenging.

File: readme.md (hunk 1 above)
Comment (NNNILabs) [reply readme.md]: Done, changes made.

**Comment by jaromir-sukuba** [comment]
Thank you for readme language corrections.

**Pull request merged**